tensorflow_io/core/python/experimental/avro/python/README.md

```diff
@@ -0,0 +1,8 @@
+This folder only contains the tests.
```

Comment:
Is it necessary to have additional python modules `tensorflow_io/core/python/experimental/avro/python` and `tensorflow_io/core/python/experimental/avro/python/util`?

I think you can move `README.md`, and `avro_serialization.py` to `tensorflow_io/core/python/experimental/avro/`,  and delete all directories that does not have a concrete python source file.

Reply:
updated